src/Proto/nodeguard.proto

```diff
@@ -88,6 +89,8 @@ message LiquidityRule {
   float rebalance_target = 7;
   optional string reverse_swap_address = 8;
   bool is_reverse_swap_wallet_rule = 9;
+  // Public key of the node that is the counterparty of the channel
+  string remote_node_pubkey = 10;
```

Comment:
do you also send it if it's not managed by us?

Reply:
Yes, they way this works is that liquidator will check both pubkeys, if they are managed, it will trigger a special rebalance. So yes, we need both pubkeys all the time.